tools/code_analyzer/run_analyzer.sh

```diff
@@ -19,8 +19,8 @@ echo "Analyze: ${INPUT}"
   -op_invoke_pattern="c10::Dispatcher::findSchema" \
   -root_symbol_pattern="torch::jit::[^(]" \
   -torch_library_init_pattern="^.*TORCH_LIBRARY_init_([^(]+)(\(.*)?$" \
-  -torch_library_init_pattern="^.*TORCH_LIBRARY_FRAGMENT_init_([^(]+)(\(.*)?$" \
-  -torch_library_init_pattern="^.*TORCH_LIBRARY_IMPL_init_([^(]+)_([^_]+)(\(.*)?$" \
+  -torch_library_init_pattern="^.*TORCH_LIBRARY_FRAGMENT_init_([^_]+)_\d*(\(.*)?$" \
+  -torch_library_init_pattern="^.*TORCH_LIBRARY_IMPL_init_([^_]+)_([^_]+)_\d*(\(.*)?$" \
```

Comment:
updating the regex to skip the extra `_<digits>` that I added to the variable names in `TORCH_LIBRARY_FRAGMENT` and `TORCH_LIBRARY_IMPL`. Note that I didn't need to make any changes to `TORCH_LIBRARY`; it already enforces that it can't be called multiple times with the same namespace, guaranteeing that we don't have any variable name collisions.

Also, I don't think that the IMPL regex previously captured the dispatch key and the function arguments in separate match groups (it looked like they ended up getting bundled together into the second group, since `[^_]` greedily matched everything from the dispatch key name through the end of the function. I don't think that the other match groups are used anywhere (so it shouldn't affect anything), but this change should get the second match group to only match on the dispatch key name.

Reply:
star and not plus, in the regex here?